pkg/sloop/store/untyped/store.go

```diff
@@ -16,21 +16,45 @@ import (
 	"time"
 )
 
-func OpenStore(factory badgerwrap.Factory, rootPath string, configPartitionDuration time.Duration) (badgerwrap.DB, error) {
+func OpenStore(factory badgerwrap.Factory, rootPath string, configPartitionDuration time.Duration, badgerMaxTableSize int64, badgerKeepL0InMemory bool, badgerVLogFileSize int64, badgerVLogMaxEntries uint, badgerUseLSMOnlyOptions bool) (badgerwrap.DB, error) {
+	if configPartitionDuration != time.Hour && configPartitionDuration != 24*time.Hour {
+		return nil, fmt.Errorf("Only hour and day partitionDurations are supported")
+	}
+
 	err := os.MkdirAll(rootPath, 0755)
 	if err != nil {
 		glog.Infof("mkdir failed with %v", err)
 	}
 	// For now using a temp name because this all need to be replaced when we add real table/partition support
-	opts := badger.DefaultOptions(rootPath)
+	var opts badger.Options
+	if badgerUseLSMOnlyOptions {
+		opts = badger.LSMOnlyOptions(rootPath)
+	} else {
+		opts = badger.DefaultOptions(rootPath)
+	}
+
+	if badgerMaxTableSize != 0 {
```

Comment:
wondering about the grouping aspects of these 3 values - does it make sense to require all 3 if any of them is set?

Reply:
I thinks its legitimate to set just one (I had to do it for testing already)